nvflare/app_common/aggregators/dxo_aggregator.py

```diff
@@ -113,7 +113,7 @@ def accept(self, dxo: DXO, contributor_name, contribution_round, fl_ctx: FLConte
         n_iter = dxo.get_meta_prop(MetaKey.NUM_STEPS_CURRENT_ROUND)
         if n_iter is None:
             if self.warning_count.get(contributor_name, 0) <= self.warning_limit:
-                self.log_warning(
+                self.log_debug(
```

Comment:
@holgerroth @IsaacYangSLA can you comment on this one?
Do we want to warn the users if we can't find `NUM_STEPS_CURRENT_ROUND`

Reply:
Yes, I would keep that warning. Maybe we can reduce the warning_limit to 3?